src/CoreBundle/Resources/config/twig.xml

```diff
@@ -18,11 +18,11 @@
         <service id="sonata.core.twig.deprecated_template_extension" class="Sonata\CoreBundle\Twig\Extension\DeprecatedTemplateExtension">
             <tag name="twig.extension"/>
         </service>
-        <service id="sonata.core.twig.template_extension" class="Sonata\CoreBundle\Twig\Extension\TemplateExtension">
+        <service id="sonata.core.twig.template_extension" class="Sonata\Twig\Extension\TemplateExtension">
             <tag name="twig.extension"/>
             <argument>%kernel.debug%</argument>
             <argument type="service" id="translator"/>
-            <argument type="service" id="sonata.core.model.adapter.chain"/>
+            <argument type="service" id="sonata.doctrine.model.adapter.chain"/>
```

Comment:
This should be fine if people don't redefine a service with `sonata.core.model.adapter.chain` as id, which we should not support.